paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -148,33 +149,40 @@ class PaymentSheet internal constructor(
     @Parcelize
     class IntentConfiguration(
```

Comment:
Can we make this internal as well?

Reply:
Nope, as this is going to be used in the public API (and is currently being used in the playground).